proto/composerapis/astria/composer/v1alpha1/generic_collector.proto

```diff
@@ -0,0 +1,19 @@
+syntax = 'proto3';
+
+package astria.composer.v1alpha1;
+
+import "google/protobuf/empty.proto";
+
+message SequenceAction {
+  string rollup_id = 1;
+  bytes tx_bytes = 2;
+}
+
+message SubmitSequenceActionsRequest {
+  // Sequence Actions the user wants to send to the sequencer app.
+  repeated SequenceAction sequence_actions = 1;
+}
+
+service ComposerService {
+  rpc SubmitSequenceActions(SubmitSequenceActionsRequest) returns (google.protobuf.Empty) {}
```

Comment:
it would be cool if this returned the tx hashes of the txs that were submitted to sequencer containing the submitted sequence actions

Reply:
we were initially thinking of such a design. But on further discussions, we chose to stuck with the design of sending the sequence actions over to the executor over a channel. 
I do believe composer will evolve a bit from here.